addons/mail/static/src/discuss/channel_member_list.js

```diff
@@ -20,6 +20,10 @@ export class ChannelMemberList extends Component {
                 this.threadService.fetchChannelMembers(nextProps.thread);
             }
         });
+        onWillRender(() => {
+            this.onlineMembers = this.props.thread.onlineMembers;
+            this.offlineMembers = this.props.thread.offlineMembers;
```

Comment:
This is not the most optimized, the getters are still doing useless loops

Reply:
It's decent enough for the perceived performance issue. I don't want to put more LOCs than necessary, especially when the suggested "most optimized" changes go in a wrong direction for code maintainability